<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tcp</class>
 <widget class="QDialog" name="tcp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>651</width>
    <height>354</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"> background-color: rgb(18, 53, 85);</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>250</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> background-color: rgb(225, 238, 210);
font: 14pt &quot;Sans Serif&quot;;
color: rgb(230, 155, 3);</string>
   </property>
   <property name="text">
    <string>连接</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_ip">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>110</y>
     <width>231</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: rgb(225, 238, 210);
color: rgb(18, 53, 85);
font: 10pt &quot;Sans Serif&quot;;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(230, 155, 3);
font: 14pt &quot;Sans Serif&quot;;</string>
   </property>
   <property name="text">
    <string>目标ip：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(230, 155, 3);
font: 14pt &quot;Sans Serif&quot;;</string>
   </property>
   <property name="text">
    <string>目标端口：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_port">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>180</y>
     <width>231</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(225, 238, 210);
color: rgb(18, 53, 85);
font: 10pt &quot;Sans Serif&quot;;</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>100</y>
     <width>151</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(225, 238, 210);
color: rgb(18, 53, 85);
font: 10pt &quot;Sans Serif&quot;;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>250</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> background-color: rgb(225, 238, 210);
font: 14pt &quot;Sans Serif&quot;;
color: rgb(230, 155, 3);</string>
   </property>
   <property name="text">
    <string>开始点餐</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>290</y>
     <width>651</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(230, 155, 3);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>30</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(230, 155, 3);
font: 17pt &quot;Sans Serif&quot;;</string>
   </property>
   <property name="text">
    <string>欢迎光临</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
